test_helm_install_uninstall.py

```diff
@@ -0,0 +1,140 @@
+import pytest
+import subprocess
+from time import sleep
+import hpe_3par_kubernetes_manager as manager
+import logging
+
+timeout = 900
+
+def test_helm_uninstall():
+    try:
+        node_obj = manager.hpe_list_node_objects()
+        node_list = node_obj.items
+        # Fetching master node ip
+        for item in node_list:
+            if item.spec.taints != "None":
+                host_ip = item.status.addresses[0].address
+                host_name = item.status.addresses[1].address
+            break
+
+        # ssh to master node and check csi driver installed
+        command = "helm ls -n kube-system"
+        command_output = manager.get_command_output(host_name, command)
+        if len(command_output) == 1:
+            print("Driver not installed")
+            return
+
+        # ssh to master node and execute uninstall commands
+        command = "helm uninstall hpe-csi --namespace kube-system"
+        command_output = manager.get_command_output(host_name, command)
+        assert command_output[0] == 'release "hpe-csi" uninstalled', "Uninstall of 'hpe-csi' driver failed"
+        sleep(30)
+
+        # Delete crds
+        crd_obj = manager.hpe_list_crds()
+        for crd in crd_obj:
+            manager.hpe_delete_crd(crd)   
+        sleep(30) 
```

Comment:
Instead of having sleep, polling can be done to check if CRDs are deleted. This prevents test failure if system fails to delete crd due to some unknown reason, and also we save time if CRDs get deleted before 30 secs. 
Polling implementation is done in check_if_deleted().
Or check_if_deleted() can be expanded to accommodate CRDs.